packages/playground/website/vite.config.ts

```diff
@@ -133,6 +133,24 @@ export default defineConfig(({ command, mode }) => {
 				outputFile: 'assets-required-for-offline-mode.json',
 				distDirectoriesToList: ['./', '../remote', '../client'],
 			}) as Plugin,
+
```

Comment:
Indirectly related. The builder/index.php file was not being included in the website builds, and that is the file that showed me we don't need to be setting PHP files aside at all because `custom-redirects.php` is already invoked for all requests handled with PHP.